src/screens/AvatarBuilder.js

```diff
@@ -118,22 +124,29 @@ const AvatarBuilder = ({ route: { params } }) => {
           paddingTop={colorCircleTopPadding + 7}
           width="100%"
         >
-          <Animated.View
-            alignSelf="center"
-            borderColor={currentAccountColor}
-            borderRadius={19}
-            borderWidth={3}
-            height={38}
-            position="absolute"
-            style={{
-              transform: [{ translateX }],
+          <ScrollableColorPicker
+            contentOffset={{
+              x: params.initialAccountColor * 39,
             }}
-            top={colorCircleTopPadding}
-            width={38}
-          />
-          {avatarColors}
+            horizontal
+            showsHorizontalScrollIndicator={false}
+          >
+            <Animated.View
```

Comment:
Can you use the styled component here? I see a lot of attributes that are static